test/test_helper.rb

```diff
@@ -17,4 +17,63 @@ def after_teardown
     DatabaseCleaner.clean
     super
   end
+
+  def permit(current_user, record, action)
+    self.class.to_s.gsub(/Test/, '')
+        .constantize.new(current_user, record)
+        .public_send("#{action}?")
+  end
```

Comment:
no, but it can be just the variable action, because I'm passing 'new?' from the policy tests. now it is:
```ruby
def permit(current_user, record, action)
    self.class.to_s.gsub(/Test/, '')
        .constantize.new(current_user, record)
        .public_send(action)
end
```